whois-api/src/test/java/net/ripe/db/whois/api/log/JettyRequestLogTestIntegration.java

```diff
@@ -92,6 +93,18 @@ public void log_request_x_forwarded_for() throws Exception {
     }
 
 
+    @Test
+    public void password_filtered() throws Exception {
+        RestTest.target(getPort(), "whois/test/person/TP1-TEST?password=some-api_key-123")
```

Comment:
Please test some other cases:
* Multiple password query parameters
* Mixed password and other query parameters
Thanks

Reply:
Thanks for adding the additional tests